doc/cdi-operator-olm.md

```diff
@@ -0,0 +1,26 @@
+# OLM intergartion
+##OLM Overview
+https://github.com/kubevirt/kubevirt/blob/master/docs/devel/olm-integration.md
+
+
+## CDI OLM manifests 
+1. in order to obtain olm bundle manifest run the following command
+```bash
+DOCKER_REPO=<repo> DOCKER_TAG=<docker tag> PULL_POLICY=<pull policy> VERBOSITY=<verbosity> CSV_VERSION=<CSV version> QUAY_NAMESPACE=<namespace> QUAY_REPOSITORY=<application name> make manifests
+```
+The generated final olm manifests will be located in _out/manifests/release/olm/bundle/_ directory
```

Comment:
Curious about reasoning behind new manifest output dir?  Also, I suspect that our scripts that create releases and publish manifests will have to be updated to account for this.  Is that right @awels?

Reply:
We would just have to update .travis.yml to look in the new directory, shouldn't be a big deal.

Reply:
updated